### PR TITLE
Footer - transaction variant

### LINKED ISSUE #85: Footer variant for funnel (eQ)
Implement the reduced footer for within the transaction flow of eQ.

### PR DESCRIPTION
**Pull request by armstrongb**
### What is the context of this PR?
Addition of the reduced footer for use in a transactional user flow.

### How to review 
Review the branch and view the footer component.

#### Resolves
#85 Footer variant for funnel (eQ)

### CODE REVIEW

File: components/footer/_footer.scss [hunk 1]
```diff
@@ -34,3 +34,16 @@
     }
   }
 }
+.footer--eq {
+  .list--bare {
```
Comment (boxadesign) [inline components/footer/_footer.scss]: We shouldn't bring classes in from other `.scss` modules, it makes it difficult to maintain. If we need to do something different if the link is in a list, create a modifier on the link i.e. `.footer__link--list`

Comment (armstrongb) [reply components/footer/_footer.scss]: Should be resolved in the latest commit.

File: components/footer/_footer.scss [hunk 2]
```diff
@@ -34,3 +34,18 @@
     }
   }
 }
+.footer--eq {
+    .footer__link {
+      float: left;
```
Comment (bennpearson) [inline components/footer/_footer.scss]: This should be a modifier or a utility class on the `<li>` 
Either `class="footer__list--inline"` or `class="u-inline"`

File: components/footer/_footer.scss [hunk 3]
```diff
@@ -34,3 +34,18 @@
...
+      margin-right: 1.5rem;
+    }
+  .footer__license {
+      margin-top: 1.5rem;
+      border-top: 1px solid $color-borders;
+      padding-top: 1.5rem;
+      .footer__link {
```
Comment (bennpearson) [inline components/footer/_footer.scss]: This should be a modifier or a utility class on the `<li>` 
Either `class="footer__list--inline"` or `class="u-inline"`

File: components/footer/_footer.scss [hunk 4]
```diff
@@ -34,3 +34,18 @@
     }
   }
 }
+.footer--eq {
+    .footer__link {
+      float: left;
+      margin-right: 1.5rem;
+    }
+  .footer__license {
```
Comment (bennpearson) [inline components/footer/_footer.scss]: This should be a modifier or a utility class:
Either `class="footer__license--border"` or `class="u-border-top"`

**Comment by armstrongb** [comment]
I think these have been resolved now @bennpearson, @boxadesign ...

**Comment by boxadesign** [comment]
@armstrongb Yes but I'm unable to approve.

**Comment by will-madebymoon** [comment]
A margin and some padding needs to be added to the titles within the pattern library. See title'
Transactional'.

**Comment by armstrongb** [comment]
@will-madebymoon they may also need some visual treatment to clearly differentiate them from ’real‘ headings.

**Comment by armstrongb** [comment]
@bennpearson can you approve if your comments have been resolved?

**Comment by bennpearson** [comment]
@armstrongb don't think this has been resolved. don't think we need `.footer--eq` here as it's not modifying `.footer`.

File: components/footer/_footer.scss [hunk 5]
```diff
@@ -11,6 +11,16 @@
 
   .footer__license {
     margin-top: 2rem;
+    .footer__link {
```
Comment (bennpearson) [inline components/footer/_footer.scss]: No need for this class

**Pull request merged**